addons/hr_expense/tests/test_expenses_mail_import.py

```diff
@@ -59,6 +59,7 @@ def test_import_expense_from_email_product_no_cost(self):
             'standard_price': 0.0,
             'can_be_expensed': True,
             'default_code': 'product_no_cost',
+            'company_id': self.company_data['company'].id,
```

Comment:
instead, don't we wanna set a default company on product_template? @chklop 

 feels like a lot of useless diff

Reply:
By default a product should be create on all companies and not restricted to one, I don't want to change that behaviour in Odoo.